app/main/controllers/activity/RTMediaBuddyPressActivity.php

```diff
@@ -108,7 +108,8 @@ public function bp_activity_truncate_entry( $excerpt, $text, $readmore ) {
 		$dom = new DOMDocument();
 		// DOMDocument gives error on html5 tags, so we need to disable errors.
 		libxml_use_internal_errors( true );
-		$dom->loadHTML( $text );
+		// Use utf-8 encoding for different languages.
+		$dom->loadHTML( '<?xml encoding="utf-8" ?>' . $text );
```

Comment:
```suggestion
		$dom->loadHTML( '<?xml encoding="utf-8" ?>'  );
		$dom->loadHTML($text);
```
Can we write it like ☝️

Reply:
Doesn't work as expected. I think we can loadHTML only once.